js/runner_test.go

```diff
@@ -681,7 +682,8 @@ func TestVUIntegrationInsecureRequests(t *testing.T) {
 					}
 					err = vu.RunOnce(context.Background())
 					if data.errMsg != "" {
-						assert.EqualError(t, err, data.errMsg)
+						require.NotNil(t, err)
```

Comment:
can you change `require.NotNil` to `require.Error` where it's about errors - it looks nicer especially when there are  `NoError`s around

Reply:
Fixed